docs/source/getting-started.rst

```diff
@@ -264,8 +264,7 @@ duration.
     private final Timer responses = metrics.timer(name(RequestHandler.class, "responses"));
 
     public String handleRequest(Request request, Response response) {
-        final Timer.Context context = responses.time();
-        try {
+        try(final Timer.Context context = responses.time()) {
```

Comment:
You don't require the finally block, since you are using try-with-resources

Reply:
@dennyac at some point context needs to stopped for timer. 
So I think it should be in either catch block or finally, right?

Reply:
It will be closed automatically. Please read up on [try-with-resource](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) and the [AutoCloseable](https://docs.oracle.com/javase/8/docs/api/java/lang/AutoCloseable.html) interface

Reply:
Thanks @dennyac. I have updated the PR. COuld you please check now?